### PR TITLE
chore:update styling of viewing room artist link

### PR DESCRIPTION
**Pull request by nspinazz89**
The type of this PR is: Chore

This PR solves [PX-5441]

Paired with @lilyfromseattle on this

### Description
This PR updates the styling on the Viewing Room artist link to be more in line with our styling of the artist link elsewhere. 
https://www.loom.com/share/552b8bbfc2e7470589d3b013bf1a81ff

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PX-5441]: https://artsyproduct.atlassian.net/browse/PX-5441?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Mitch-Henson — APPROVED** [review]
A tiny nitpick is that an image on the PR would be helpful for frontend / style changes but otherwise it LGTM!

🚢

**Comment by nspinazz89** [comment]
> A tiny nitpick is that an image on the PR would be helpful for frontend / style changes but otherwise it LGTM!
> 
> 🚢

I put the loom video so that you could see the hover change, is it not visible to you below the description 
image

**Comment by Mitch-Henson** [comment]
@nspinazz89 I completely missed the fact you had a loom linked 🤦  Ignore me!

**Review by ansor4 — APPROVED** [review]
We definitely need to check the other pages and components that use `Details.tsx`, but otherwise 👍 

Some minor feedback, but mostly that we should either refactor the styled component, or ticket that as follow up work.

**Comment by ansor4** [comment]
@nspinazz89 assigning you to merge as you see fit, depending on if you want to include the styled component refactor work in this PR or not.

File: src/Components/Artwork/Details.tsx [hunk 1]
```diff
@@ -39,16 +34,30 @@ interface DetailsProps {
 
 const ConditionalLink: React.FC<
   Pick<DetailsProps, "includeLinks"> &
-    LinkProps &
+    RouterLinkProps &
     React.AnchorHTMLAttributes<HTMLAnchorElement>
 > = ({ includeLinks, children, ...rest }) => {
-  return includeLinks ? <Link {...rest}>{children}</Link> : <>{children}</>
+  return includeLinks ? (
+    <RouterLink {...rest}>{children}</RouterLink>
```
[the remark this answers is not in the document]
Comment (nspinazz89) [reply src/Components/Artwork/Details.tsx:41]: Ah good point I should have called that out as that is something @lilyfromseattle and I discovered when we were doing this.

File: src/Components/Artwork/Details.tsx [hunk 2]
```diff
@@ -39,16 +34,30 @@ interface DetailsProps {
...
+  ) : (
+    <>{children}</>
+  )
 }
 
+const StyledConditionalLink = styled(ConditionalLink)`
```
[the remark this answers is not in the document]
Comment (nspinazz89) [reply src/Components/Artwork/Details.tsx:47]: Awesome sounds good will ticket that and knock it out as well!

**Pull request merged**